src/ui/mucwin.c

```diff
@@ -988,3 +988,25 @@ _mucwin_set_last_message(ProfMucWin* mucwin, const char* const id, const char* c
     free(mucwin->last_msg_id);
     mucwin->last_msg_id = strdup(id);
 }
+
+gboolean
+mucwin_set_title(const char* const room_jid, const char* const disco_name)
+{
+    ProfMucWin* win = wins_get_muc(room_jid);
+    if (win) {
```

Comment:
Maybe just write error in log and return FALSE if !win? It would be more flexible approach